website/docs/pact_broker/advanced_topics/provider_verification_badges.md

```diff
@@ -6,19 +6,19 @@ _See the the_ [_Badges_](/pact_broker/configuration/features#badges) _section of
 
 If you are publishing [Provider verification results](provider_verification_results.md) to your Pact Broker \(v2.3.0+\), you can also display the verification status in your READMEs using a shiny badge like this one:
 
-[![Pact Status](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test/badge)](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test)
+[![Pact Status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/prod/badge)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/production)
```

Comment:
For branches (can i merge badge), For environments (can-i-deploy badge)

Reply:
Done - thanks!